src/api/language.js

```diff
@@ -24,6 +29,11 @@ const loadMessages = (lang) => {
     .then((messages) => Object.assign.apply({}, messages))
 }
 
+const loadSupportedLanguageMessages = () => {
+  return { en, fr, es, hi, pt }
```

Comment:
From the latest conversation about language support, it sounds like we want to allow reading in all languages even if we know some of them will look broken. It means we have to load all `i18n/*.json` into the banana instance.

Reply:
as far as I understood, or maybe wrong, that's what `banana-i18n-loader` can do, right?

Reply:
Okay yea, loading all `i18n/*.json` should help with the refresh issue. And then yes, question is should still load all of them using `banana-i18n-loader` or using `fetch`. 

With `banana-i18n-loader` we would have a long series of imports in language.js, with fetch we could have something similar to commit e6f49b183f1fec7c87f132c8cd80e0a45975be20. Is there a better way?